### ISSUE TITLE
Is GWC limited to single thread execution?

### ISSUE THREAD

**Issue by kriegaex** [groovy-console/groovy-web-console#13]
When I tried to run [this](https://gwc-experiment.appspot.com/?g=groovy_2_5&gist=702a14cf819aa5f26c8b613c574425bd) in GWC with an embedded spec runner and a custom runner configuration, the tests still ran sequentially in a single thread, while I was expecting a multi-threaded run. I think there is a limitation on the server concerning how many cores or threads GWC can use, isn't there? I guess that might make sense in order to avoid the worst of DoS attacks. I was just surprised.

If in fact this limitation was not deliberate, do you see any chance to lift it?

**Comment by kriegaex** [comment]
The document says that the 8G tier has 2 vCPU. However, I have no idea what that actually means with regard to being able to use multiple threads. As it seems, there is some limitation. I never used any cloud computing stuff, so I cannot say for sure. But if you say that there is nothing that can or should be done because of technical limitations or pricing issues, feel free to simply close the ticket with a final comment. It is not important, I just noticed that it did not work when I wanted to post a GWC example for multi-threading on Stack Overflow.

**Comment by leonard84** [comment]
I don't think that we can do anything here, the `2vCPU` just means the computing equivalent of `1 vCPU is equal to 2.4GHz` so it could be achieved by one core with `4.8GHz`.
